app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+    @user = @post.author
+  end
+
+  def create
+    @comment = Comment.new(post_params)
+    @comment.author = current_user
+    @comment.post = Post.find(params[:post_id])
+    if @comment.save
+      redirect_to user_post_path(@comment.post.author, @comment.post)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def post_params
```

Comment:
**OPTIONAL** 
- [ ] Please rename your params method to `comment_params` since it's found in the CommentController.
### Remember to refactor your create method